File: stkpush/src/main/java/com/freddygenicho/mpesa/stkpush/api/response/Item.java
package com.freddygenicho.mpesa.stkpush.api.response;

/**
 * @author Fredrick Ochieng on 02/02/2018.
 */

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Item {

    @SerializedName("Name")
    @Expose
    private String name;
    @SerializedName("Value")
    @Expose
    private Integer value;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getValue() {
        return value;
    }

    public void setValue(Integer value) {
        this.value = value;
    }

}

File: stkpush/src/main/java/com/freddygenicho/mpesa/stkpush/api/response/Item.kt
package com.freddygenicho.mpesa.stkpush.api.response

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

/**
 * @author Fredrick Ochieng on 02/02/2018.
 */
class Item {
    @SerializedName("Name")
    @Expose
    var name: String? = null

    @SerializedName("Value")
    @Expose
    var value: Int? = null
}